packages/sandboxd/spec/admin/admin.proto

```
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+package sandboxd.admin.v1;
```

Buf (1.70.0)

[error] Files with package "sandboxd.admin.v1" must be within a directory "sandboxd/admin/v1" relative to root but were in directory "packages/sandboxd/spec/admin".

(PACKAGE_DIRECTORY_MATCH)